tasks/notify.py

```diff
@@ -459,40 +460,58 @@ def send_notification(ctx: Context, alert_jobs):
 
 
 @task
-def send_failure_summary_notification(_, jobs: dict[str, any] | None = None, list_max_len=10):
+def send_failure_summary_notification(
+    _, jobs: dict[str, any] | None = None, list_max_len=10, jobowners=".gitlab/JOBOWNERS"
+):
+    from slack_sdk import WebClient
+
     if jobs is None:
         jobs = os.environ["JOB_FAILURES"]
         jobs = json.loads(jobs)
 
     # List of (job_name, (failure_count, total_count)) ordered by failure_count
     stats = sorted(
-        ((name, (fail, success)) for (name, (fail, success)) in jobs.items() if fail > 0),
+        ((name, (fail, total)) for (name, (fail, total)) in jobs.items() if fail > 0),
         key=lambda x: (x[1][0], x[1][1] if x[1][1] is not None else 0),
         reverse=True,
-    )[:list_max_len]
+    )
 
     # Don't send message if no failure
     if len(stats) == 0:
         return
 
-    # Create message
-    message = ['*Daily Job Failure Report*']
-    message.append('These jobs had the most failures in the last 24 hours:')
-    for name, (fail, total) in stats:
-        link = get_ci_visibility_job_url(name)
-        message.append(f"- <{link}|{name}>: *{fail} failures*{f' / {total} runs' if total else ''}")
+    # Partitionate by channels as some teams share the same slack channel (avoid duplicate messages)
```

Comment:
```suggestion
    # Partition by channels as some teams share the same slack channel (avoid duplicate messages)
```

I think "Partitionate" [does not exist](https://www.oed.com/search/dictionary/?scope=Entries&q=Partitionate) in English, but "Partition" does.